hydrolib/core/dflowfm/ini/models.py

```diff
@@ -55,6 +56,15 @@ class Config:
         extra = Extra.ignore
         arbitrary_types_allowed = False
 
+    def __init__(self, **data):
+        super().__init__(**data)
+        self._unknown_keyword_error_manager.raise_error_for_unknown_keywords(
+            data,
```

Comment:
It seems we are only using the keys of this dict in the `raise_error_for_unknown_keywords()` method. So just pass `data.keys()` and update the types expected by the method as you will now pass a list of strings instead of a dict.

Reply:
Nevermind, ignore this comment. It seems perfectly fine to just pass the dict :)

Reply:
I will ignore this comment :)